src/Views/WhatsNewPage.xaml

```diff
@@ -279,7 +280,6 @@
                                     <Grid.RowDefinitions>
                                         <RowDefinition Height="Auto" />
                                         <RowDefinition Height="*" />

```

Comment:
Why did you choose WrapPanel? I think that's usually used for items that are supposed to be displayed horizontally, but wrap if necessary. These items are never horizontal, right? I think the much simpler solution is just to change the "*" on this line to "Auto".

In any case, I think we have an accessibility issue here where if you have text scaling way up, the text gets cut off. We'll have to turn these into ScrollViewers eventually. But I'd just go with fixing the immediate problem for now.

Reply:
I choose the wrap panel because changing the "*" to "Auto" results in this:
<img width="214" alt="image" src="https://github.com/microsoft/devhome/assets/13912953/470ef771-a3e1-42c0-9cf2-4a121373b2e7">
The text clips instead of wrapping nicely as it get hidden behind the button.

The designers wanted the text to wrap with ellipsis when out of space, so using the wrap panel with vertical orientation preserves the wrapping behavior while aligning the link just below the text.
<img width="215" alt="image" src="https://github.com/microsoft/devhome/assets/13912953/f746d84c-f38a-466f-bbd7-4b209dca900e">

Reply:
Ok. Sounds like Narrator still reads the cut off text, so not an accessibility bug. Different issue, but there *is* an accessibility bug that it thinks there's a link on every card, not just the Dev Drive one. It reads "Link" but not any text/description for the link, which is confusing.